.github/workflows/release-service-stream.yml

```diff
@@ -10,7 +10,7 @@ env:
   DOCKERHUB_USER: "keboolabot"
   DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
 
-  IMAGE_TAG: ${{ github.ref_name }}
+  IMAGE_TAG: production-${{ github.ref_name }}
```

Comment:
Maybe it would be better for us to actually use the prefix when tagging? This way we don't have the option to create a `dev-*` image.

Reply:
Some production samples:
`apps-proxy-v1.1.0`
`stream-v0.3.17`
These are dev samples
`apps-proxy-v0.3.0-test.3`
We used suffix instead of prefix for deployment only to development. We found this convinient as the semantic versioning counts on this approach to suffix it with any characters.